dxf2png/README.md

```diff
@@ -0,0 +1,16 @@
+# DXF to PNG Converter
```

Comment:
We can remote this `README.md` and merge the relevant information in the top level `README.md`

Reply:
This one is done!